plugin/trino-clickhouse/src/main/java/io/trino/plugin/clickhouse/ClickHouseClient.java

```diff
@@ -95,6 +95,7 @@
 import java.util.concurrent.atomic.AtomicReference;
 import java.util.function.BiFunction;
 
+import static com.clickhouse.client.ClickHouseValues.convertToQuotedString;
```

Comment:
Do we have this API in the legacy version ?

Reply:
In this case can we squash with the next commit - I'm happy with both the cases.

Reply:
It existed, but the package was changed in the new version.